lib/Doctrine/Common/Annotations/TokenParser.php

```diff
@@ -82,7 +93,11 @@ public function next($docCommentIsComment = TRUE)
      */
     public function parseUseStatement()
     {
+        return PHP_VERSION_ID >=80000 ? $this->parseUseStatementV8() : $this->parseUseStatementV7();
```

Comment:
Please use a fully qualified name for the `PHP_VERSION_ID` constant, so that PHP can resolve that condition at compile time.